languages/rust/exercises/concept/tuples/.docs/example.rs

```diff
@@ -0,0 +1,28 @@
+use std::fmt::Display;
+const TODAYS_CHOICE: usize = 1;
+/// get today's prizes
+fn prizes() -> (usize, char, String, i32, bool, f32, ()) {
+    (0, '🥚', "🥚🥚".to_string(), 0, false, 1.5, ())
+}
+/// handle an attendee's guess.
+fn win_or_lose(guess: usize) -> Option<impl Display> {
+    if guess == TODAYS_CHOICE {
+        let winning = prizes();
+        if guess == 0 {
+            Some(winning.0)
+        } else {
+            Some(winning.1)
+        }
+    } else {
+        None
+    }
+}
+#[cfg(test)]
+mod tests {
+    use super::*;
+    #[test]
+    fn get_prizes() {
+        let list = prizes();
+        assert_eq!(list.0, 0);
+    }
+}
```

Comment:
Out of interest, what is this? Is this something the student would be expected to write, or just something you've put in this file for now to demonstrate something?

Reply:
👋 Rust let's use incorporate test modules within a Rust file. `cargo new --lib my-project` generates the boilerplate. For now I am using it as a sanity check for asserting the `example.rs` logic does what I need. Upon approval of the given Rust logic I hope to migrate these into the full test suite like the other concept exercises.

Reply:
Awesome. Thanks for explaining. I'm going to adopt the same approach in my Ruby ones.